### ISSUE TITLE
Prep for wider release

### ISSUE THREAD

**Issue by robbyoconnor** [praece/sails-bookshelf#1]
Just curious if this is useable yet?

**Comment by kevinob11** [comment]
I'm using it in production on an app, and it works well but...
- Super opinionated to my current project
  - Built in cleaning of model params on save based on config attributes
  - Nested save and update based on my use case
  - Hard coded id and createdAt / updatedAt keys
- The model config syntax is a little meh
- I haven't moved the tests from my app to here yet

That being said it does have all of the following!
- Built in blueprints that mirror the standard sails query syntax in bookshelf
- Populating and querying of relations via blueprint params
- Basic and custom validation
- Nested save that uses transactions to properly handle failures

I'd love to get it cleaned up and released at some point, but realistically it will likely be early April before I can spend any time on that.

**Comment by robbyoconnor** [comment]
:heart:

**Comment by robbyoconnor** [comment]
How do I use it :( By the way great work :)

**Comment by kevinob11** [comment]
I might be able to at least write a better Readme tomorrow, but here is the gist of it:

Disable the waterline ORM in `.sailsrc`

```
{
  "generators": {
    "modules": {}
  },
  "hooks": {
    "orm": false,
    "pubsub": false
  }
}
```

Set your connection settings in `config/connection.js`

```
module.exports.connection = {
  client: 'postgresql',
  connection: postgres://user:password@hostname:5432/database,
  migrations: {
    tableName: 'db_migrations',
    directory: 'db/migrations'
  },
  seeds: {
    directory: 'db/seeds'
  }
};
```

Then set up a model or two based on my example in the readme. You can use all of the same post/get/put/delete syntax that ships with waterline. For custom controllers all of the standard bookshelf syntax works as expected. Feel free to hit me up here if you have any questions.

**Comment by kevinob11** [comment]
Also don't actually put your postgres connection url in connection.js, put it in local.js and keep it out of your git repo :)

**Comment by robbyoconnor** [comment]
@kevinob11 -- glad I ran into you on that issue! Thank you so much! You may close this if you wish.

**Comment by kevinob11** [comment]
No problem, I'm going to leave it open for now, I'll change the name and close it when I clean it up.

**Comment by robbyoconnor** [comment]
Do you have a sample project?

**Comment by robbyoconnor** [comment]
I think I got it -- nevermind :heart: is there a slack somewhere?

**Comment by kevinob11** [comment]
Nope, but you're welcome to email me kevin.obrien@praece.com if you need something. Won't always get back to you right away, but worth a shot.
